qlty-cli/src/commands/githooks/install.rs

```diff
@@ -0,0 +1,109 @@
+use crate::{Arguments, CommandError, CommandSuccess};
+use anyhow::{Context as _, Result};
+use clap::Args;
+use qlty_config::Workspace;
+use std::fs;
+use std::path::Path;
+
+#[cfg(unix)]
+use std::os::unix::fs::PermissionsExt as _;
+
+#[derive(Args, Debug)]
+pub struct Install {}
+
+const QLTY_HOOKS_DIR: &str = ".qlty/hooks";
+
+const PRE_COMMIT_HOOK: &str = r#"#!/bin/sh
+qlty fmt --trigger pre-commit --index-file="$GIT_INDEX_FILE"
+"#;
```

Comment:
@brynary FYI something to consider here is using the Rust includr_str embed approach from #1262 to separate this out into its own .sh file so that we can have shellcheck run against the file, more easily edit, etc.